packages/ui/components/input-file/src/validators.js

```diff
@@ -0,0 +1,110 @@
+import { Validator } from '@lion/ui/form-core.js';
+import { getLocalizeManager } from '@lion/ui/localize-no-side-effects.js';
+
+/**
+ * @typedef {import('../../form-core/types/validate/validate.js').ValidatorConfig} ValidatorConfig
+ * @typedef {import('../../form-core/types/validate/validate.js').ValidatorParam} ValidatorParam
+ * @typedef {import('../../form-core/types/validate/validate.js').ValidatorOutcome} ValidatorOutcome
+ * @typedef {import('../types/input-file.js').InputFile} InputFile
+ */
+
+/* eslint max-classes-per-file: ["error", 2] */
+export class FileValidation extends Validator {
+  static get validatorName() {
+    return 'FileValidation';
+  }
+
+  /**
+   * Syntactic sugar for getting the file extension
+   * @param {typeof File.name} fileName
+   * @return {string}
+   */
+  // eslint-disable-next-line class-methods-use-this
+  _getExtension = fileName => fileName?.slice(fileName.lastIndexOf('.'));
```

Comment:
This should be `_getExtension = fileName => fileName?.slice(fileName.lastIndexOf('.') + 1)` since it now compares for example `.txt` to `txt`.

Reply:
fixed